src/api/app/controllers/webui/staging_workflows/request_ignorations_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Webui
+  module StagingWorkflows
```

Comment:
Please note that we have now a ``Staging`` namespace introduced in https://github.com/openSUSE/open-build-service/pull/6237

So you should group it into: ``Webui::Staging::``

Several other reviewers already pointed out that the naming does not make much sense. With the namespace what about

```ruby
WebUI::Staging::Ignored::RequestsController
WebUI::Staging::Staged::RequestsController
etc
```

Reply:
> Please note that we have now a `Staging` namespace introduced in #6237
> 
> So you should group it into: `Webui::Staging::`
> 
> Several other reviewers already pointed out that the naming does not make much sense. With the namespace what about
> 
> ```ruby
> WebUI::Staging::Ignored::RequestsController
> WebUI::Staging::Staged::RequestsController
> etc
> ```

After applying the changes suggested in [this thread](https://github.com/openSUSE/open-build-service/pull/6201#pullrequestreview-174787082) the controller was moved to:

`app/controllers/webui/staging/excluded_requests_controller.rb`

The namespace is

`Webui::Staging::ExcludedRequestsController`